benchmark/config/qrcode_wechatqrcode.yaml

```diff
@@ -4,18 +4,19 @@ Benchmark:
     path: "benchmark/data/qrcode"
     files: ["opencv.png", "opencv_zoo.png"]
   metric:
+    type: "Detection"
     sizes:
       - [100, 100]
       - [300, 300]
-    warmup: 3
+    warmup: 30
     repeat: 10
     reduction: "median"
   backend: "default"
   target: "cpu"
 
 Model:
   name: "WeChatQRCode"
-  detect_prototxt_path: "models/qrcode_wechatqrcode/detect_2021sep.prototxt"
-  detect_model_path: "models/qrcode_wechatqrcode/detect_2021sep.caffemodel"
-  sr_prototxt_path: "models/qrcode_wechatqrcode/sr_2021sep.prototxt"
-  sr_model_path: "models/qrcode_wechatqrcode/sr_2021sep.caffemodel"
\ No newline at end of file
+  detect_prototxt_path: "models/qrcode_wechatqrcode/detect_2021nov.prototxt"
+  detect_model_path: "models/qrcode_wechatqrcode/detect_2021nov.caffemodel"
+  sr_prototxt_path: "models/qrcode_wechatqrcode/sr_2021nov.prototxt"
+  sr_model_path: "models/qrcode_wechatqrcode/sr_2021nov.caffemodel"
```

Comment:
@fengyuentau how do you define the suffix? Isn't QRCode models trained quite some time ago?

Reply:
The suffix is the time I uploaded to zoo. Previous was wrong.